File: med-app-backend-spring/src/main/java/com/medapp/app/dts/medappbackendspring/Dto/DoctorMainInfo.java
package com.medapp.app.dts.medappbackendspring.Dto;

import lombok.*;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class DoctorMainInfo {
    private Long id;
    private String email;
    private String firstname;
    private String lastname;
    private String phoneNumber;
    private String description;
    private Double rating;
    private String city;
    private List<FeedbackDto> receivedFeedbacks;

}

File: med-app-android/app/src/main/java/dts/app/med_app_android/Model/DoctorMainInfo.kt
package dts.app.med_app_android.Model

data class DoctorMainInfo(
    val id: Long,
    val email: String,
    val firstname: String,
    val lastname: String,
    val phoneNumber: String,
    val description: String,
    val city: String,
    val rating: Double,
    val receivedFeedbacks: List<FeedbackDto>
)